src/languages/es.ts

```diff
@@ -1375,35 +1375,35 @@ export default {
         headerTitle: 'Condiciones y tarifas',
         haveReadAndAgree: 'He leído y acepto recibir ',
         electronicDisclosures: 'divulgaciones electrónicas',
-        agreeToThe: 'Estoy de acuerdo con la ',
-        walletAgreement: 'Acuerdo de billetera',
+        agreeToThe: 'Estoy de acuerdo con el ',
+        walletAgreement: 'Acuerdo de la billetera',
         enablePayments: 'Habilitar pagos',
-        feeAmountZero: '$0',
+        feeAmountZero: 'US$0',
```

Comment:
Did not know we had this. IMO this is wrong and should not be in this file. We should be using `numberFormat` for this